syntax = "proto2";
import "session_commands.proto";

message Command_ReplayDownload {
    extend SessionCommand {
        optional Command_ReplayDownload ext = 1101;
    }
    optional sint32 replay_id = 1 [default = -1];
}
